linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_unix2_chkpwd/tests/ocp4/e2e.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
This file is not needed because the rule is not applicable for OCP4.

Reply:
File dropped at 73b5042